e2e/helper.go

```diff
@@ -28,7 +28,7 @@ func NewContainer(image string, privatePort int) *Container {
 
 // Start starts a new docker container on a random port
 func (c *Container) Start(t *testing.T) {
-	result := icmd.RunCommand("docker", "run", "--rm", "-d", "-P", c.image).Assert(t, icmd.Success)
+	result := icmd.RunCommand("docker", "run", "--rm", "--privileged", "-d", "-P", c.image).Assert(t, icmd.Success)
```

Comment:
Why is --privileged used here ? Do we use DinD or something similar ?

Reply:
Just read the code, ok, we use DinD (and I suppose we'll run Kind for k8s deployment tests as well)

Reply:
The idea is not to test the stack deployment itself, which should be tested in the docker cli, but all the cycle from `docker-app` binary to the `cnab-run` backend.